src/gui/editors/AutomationEditor.cpp

```diff
@@ -1119,22 +1008,33 @@ inline void AutomationEditor::drawCross( QPainter & p )
 
 
 
-inline void AutomationEditor::drawAutomationPoint( QPainter & p, timeMap::iterator it )
+inline void AutomationEditor::drawAutomationPoint(QPainter & p, timeMap::iterator it)
 {
-	int x = xCoordOfTick( it.key() );
-	int y = yCoordOfLevel( it.value() );
-	const int outerRadius = qBound( 3, ( m_ppb * AutomationPattern::quantization() ) / 576, 5 ); // man, getting this calculation right took forever
-	p.setPen( QPen( vertexColor().lighter( 200 ) ) );
-	p.setBrush( QBrush( vertexColor() ) );
-	p.drawEllipse( x - outerRadius, y - outerRadius, outerRadius * 2, outerRadius * 2 );
+	QMutexLocker m(&m_patternEditorMutex);
+
+	int x = xCoordOfTick(it.key());
+	int y;
+	const int outerRadius = qBound(3, (m_ppb * AutomationPattern::quantization()) / 576, 5); // man, getting this calculation right took forever
+
+	// Draws an ellipse for the outValue
```

Comment:
```suggestion
	// Draw a circle for the outValue
```
May as well be more precise, right?